site/content/docs/5.3/components/side-navigation.md

```diff
@@ -0,0 +1,1425 @@
+          <button class="side-nav-toggle collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapsibleElement3" aria-expanded="false" aria-controls="collapsibleElement3"><span class="visually-hidden">Open/Close side navigation</span></button>
+          <ul class="side-nav-content side-nav-scrollable list-unstyled m-0">
+            <li data-bs-toggle="tooltip" data-bs-title="Dashboard" data-bs-placement="right">
+              <a href="#" class="side-nav-item active fw-bold">
+                <svg width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+                  <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#dashboard"/>
+                </svg>
+                Dashboard
+              </a>
+            </li>
+            <li data-bs-toggle="tooltip" data-bs-title="Charts" data-bs-placement="right">
+              <a href="#" class="side-nav-item fw-bold">
+                <svg width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+                  <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#chart"/>
+                </svg>
+                Charts
+              </a>
+            </li>
+            <li data-bs-toggle="tooltip" data-bs-title="Financial" data-bs-placement="right">
+              <a href="#" class="side-nav-item fw-bold">
+                <svg width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+                  <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#financial"/>
+                </svg>
+                Financial
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="d-none d-lg-flex flex-grow-1 p-2">Content that should fill the remaining space.</div>
+</div>
+
+```html
+<button class="btn btn-primary d-lg-none" data-bs-target="#sideNavCollapsibleRight" data-bs-toggle="offcanvas" aria-haspopup="true" aria-controls="sideNavCollapsibleRight" id="sideNavCollapsibleRightToggle">Collapsing side navigation</button>
+
+<div class="offcanvas-lg offcanvas-end" role="region" aria-labelledby="sideNavCollapsibleRightToggle" id="sideNavCollapsibleRight">
+  ...
+</div>
+
+<div class="d-none d-lg-flex flex-grow-1 p-2">Content that should fill the remaining space.</div>
+```
+-->
+
+## Dark variant
+
+Add `.side-nav-dark` to the `.side-nav` or its children for a dark variant. For example the [collapsible side navigation with drawer content](#collapsible-with-drawer-content) is made with a part of dark variant.
+
+We strongly recommend to use `.offcanvas-dark` (resp. `.btn-close-white`) on top of `.offcanvas-*` (resp. `.btn-close`) in order to have the correct design.
+
+{{< callout info >}}
+Dark variant can be applied on root of any example except the [collapsible side navigation with drawer content](#collapsible-with-drawer-content) one.
```

Comment:
It may be a bit weird to give an example with a link to collapsible side navigation with drawer content and add a callout 5 lines after saying that it's not compatible with collapsible side navigation with drawer content.

Reply:
```suggestion
Dark variant shouldn't be applied on the root element of the [collapsible side navigation with drawer content](#collapsible-with-drawer-content) one.
```
Would it be better here ?

Reply:
Yes, I prefer this version 👍